### PR TITLE
begin refactor for TF1.0

### PR DESCRIPTION
**Pull request by normanheckscher**
Refactor for TF1.0alpha
Unable to finish multi-GPU as I don't have a machine to test it on.

### CODE REVIEW

File: notebooks/3_NeuralNetworks/bidirectional_rnn.ipynb [hunk 1]
```diff
@@ -126,109 +121,24 @@
...
-      "Iter 96000, Minibatch Loss= 0.098764, Training Accuracy= 0.96875\n",
-      "Iter 97280, Minibatch Loss= 0.115553, Training Accuracy= 0.94531\n",
-      "Iter 98560, Minibatch Loss= 0.079704, Training Accuracy= 0.96875\n",
-      "Iter 99840, Minibatch Loss= 0.064562, Training Accuracy= 0.98438\n",
-      "Optimization Finished!\n",
-      "Testing Accuracy: 0.992188\n"
-     ]
-    }
-   ],
+   "execution_count": null,
```
Comment (aymericdamien) [inline notebooks/3_NeuralNetworks/bidirectional_rnn.ipynb]: Can you add the output of the cell here?

File: notebooks/3_NeuralNetworks/convolutional_network.ipynb [hunk 2]
```diff
@@ -361,6 +185,15 @@
     "                                      y: mnist.test.labels[:256],\n",
     "                                      keep_prob: 1.})"
    ]
+  },
```
Comment (aymericdamien) [inline notebooks/3_NeuralNetworks/convolutional_network.ipynb]: Also the output of this cell.

File: notebooks/3_NeuralNetworks/recurrent_network.ipynb [hunk 3]
```diff
@@ -103,123 +98,38 @@
...
-      "Iter 96000, Minibatch Loss= 0.073954, Training Accuracy= 0.97656\n",
-      "Iter 97280, Minibatch Loss= 0.130790, Training Accuracy= 0.96094\n",
-      "Iter 98560, Minibatch Loss= 0.067689, Training Accuracy= 0.97656\n",
-      "Iter 99840, Minibatch Loss= 0.186411, Training Accuracy= 0.92188\n",
-      "Optimization Finished!\n",
-      "Testing Accuracy: 0.976562\n"
-     ]
-    }
-   ],
+   "execution_count": null,
```
Comment (aymericdamien) [inline notebooks/3_NeuralNetworks/recurrent_network.ipynb]: And this cell also.

**Comment by aymericdamien** [comment]
Thanks for refactoring :) Everything seems good now.

**Comment by normanheckscher** [comment]
No worries. 👍 

As mentioned, I haven't touched the multi-GPU code as I don't have the hardware to test it on.

**Pull request merged**